src/main/java/org/sagebionetworks/web/client/view/users/RegisterAccountViewImpl.java

```diff
@@ -1,28 +1,114 @@
 package org.sagebionetworks.web.client.view.users;
 
+import org.gwtbootstrap3.client.ui.Button;
+import org.gwtbootstrap3.client.ui.TextBox;
 import org.gwtbootstrap3.client.ui.html.Div;
+import org.sagebionetworks.web.client.DisplayConstants;
+import org.sagebionetworks.web.client.DisplayUtils;
+import org.sagebionetworks.web.client.GWTTimer;
+import org.sagebionetworks.web.client.GlobalApplicationState;
+import org.sagebionetworks.web.client.ValidationUtils;
+import org.sagebionetworks.web.client.cookie.CookieProvider;
+import org.sagebionetworks.web.client.widget.entity.controller.SynapseAlert;
+import org.sagebionetworks.web.client.widget.header.Header;
+import org.sagebionetworks.web.client.widget.login.LoginWidgetViewImpl;
+import org.sagebionetworks.web.client.widget.search.SynapseSuggestBox;
 
+import com.google.gwt.event.dom.client.KeyCodes;
+import com.google.gwt.event.dom.client.KeyDownHandler;
+import com.google.gwt.http.client.URL;
 import com.google.gwt.uibinder.client.UiBinder;
 import com.google.gwt.uibinder.client.UiField;
+import com.google.gwt.user.client.Window;
 import com.google.gwt.user.client.ui.Composite;
 import com.google.gwt.user.client.ui.Widget;
 import com.google.inject.Inject;
 
 public class RegisterAccountViewImpl extends Composite implements RegisterAccountView {
-
+	public static final String GOOGLE_OAUTH_CALLBACK_URL = LoginWidgetViewImpl.GOOGLE_OAUTH_CALLBACK_URL + "&state=";

```

Comment:
consider moving  def'n ofGOOGLE_OAUTH_CALLBACK_URL next to LoginWidgetViewImpl.GOOGLE_OAUTH_CALLBACK_URL